pytest.ini

```diff
@@ -1,6 +1,6 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=adhocracy-plus.config.settings.test
-testpaths=tests
+testpaths=tests apps/*/tests
```

Comment:
I moved the tests under the general tests folder, but we can leave this here for development purposes. And in the end we should collect all tests in the root `tests` folder,